preprocessing.py

```diff
@@ -1,31 +1,67 @@
 import librosa
 import numpy as np
 import torch
+import random
 from argparse import ArgumentParser
 
 # Preprocessing logics of Honk
 # noise are off shifting is no longer random
 # usage :
-#     ./preprocessing.py --file "data/test.wav"
+#     python ./preprocessing.py --file "data/test.wav"
 
-sampleRate = 44100
+sample_rate = 16000
+
+def print_data(name, data) :
```

Comment:
Minor comment: sometime in the future, let's make this Python 3 compatible? e.g., `from __future__ import print_function`, using `six` package, etc.

Reply:
sounds good